Gemfile

```diff
@@ -4,10 +4,9 @@ source "https://rubygems.org"
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby ">= 2.6.10"
 
-# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
-# bound in the template on Cocoapods with next React Native release.
-gem "activesupport", ">= 6.1.7.5", "< 7.1.0"
-gem "cocoapods", ">= 1.13", "< 1.15"
+# Exclude problematic versions of cocoapods and activesupport that causes build failures.
+gem "activesupport", ">= 6.1.7.5", "!= 7.1.0"
+gem "cocoapods", ">= 1.13", "!= 1.15.0", "!= 1.15.1"
```

Comment:
Does this mean we can bump up our version of cocoapods as long as it's not `1.15.0` and not `1.15.1`? If so, I'd love to see you pick an exact version above `1.15.1` & let the mobile team know which version we need to be on going forward. That would let us finally merge [this PR](https://github.com/inaturalist/iNaturalistReactNative/pull/2868) and get dependabot working again instead of ignoring all security updates.